tests/apps/query_performing_app/apps.py

```diff
@@ -0,0 +1,32 @@
+from django.apps import AppConfig
+from django.db import DatabaseError, connections
+
+
+class QueryDefaultDatabaseAppConfig(AppConfig):
+    name = "apps.query_performing_app"
+    database = "default"
+
+    def ready(self):
+        from .models import Something
+
+        try:
+            Something.objects.using(self.database).get(id=1)
+        except (DatabaseError, Something.DoesNotExist):
+            pass
```

Comment:
I had the `except DatabaseError` in here because the `Something` model table wasn't created during test setup. I'm now using the `tests.apps.models.TotallyNormal` instead because its table is somehow created...

Does this make sense? I don't really like that the `query_performing_app` now depends on a model from another app especially because `tests.apps` isn't mentioned in the `@override_settings(INSTALLED_APPS=[...])` of these tests. Should I keep using the `Something` model instead and figure out how to create its model table during test setup?

Reply:
For me, using `TotallyNormal` is fine.